_targets.R

```diff
@@ -25,29 +25,34 @@ p1_targets_list <- list(
     error = 'continue'
   ),
   tar_target(
-    site_data,
-    bind_rows(site_data_frames),
+    nwis_data_csv,
```

Comment:
You are missing `format = 'file'` for this target, though! I think what that means is if this target were rebuilt, it might not necessarily kick off downstream builds because the result of the target is just a string with the filename. So, even if the contents of the file changed, you are only tracking the file name and that is the same, so nothing downstream would happen.